paper.bib

```diff
@@ -19063,6 +19063,151 @@ @Article{Thrampoulidis2015LASSO
   timestamp   = {2018.05.18},
 }
 
+%%%%% Appending Song's References
+
+@Article{Dobra2004,
+  Title                    = {Sparse graphical models for exploring gene expression data},
+  Author                   = {Dobra, A. and Hans, C. and Jones, B. and Nevins, J. R. and Yao, G. and West, M.},
+  Journal                  = {Journal of Multivariate Analysis},
+  Year                     = {2004},
+  Number                   = {1},
+  Pages                    = {196--212},
+  Volume                   = {90},
+
+  Publisher                = {Elsevier}
+}
+
+@Unpublished{Hammersley1971,
+  Title                    = {{Markov} fields on finite graphs and lattices},
+  Author                   = {J. M. Hammersley and P. Clifford},
+  Note                     = {Unplished},
+  Year                     = {1971}
+}
+
+@InCollection{Hartemink2000,
+  Title                    = {Using graphical models and genomic expression data to statistically validate models of genetic regulatory networks},
+  Author                   = {Hartemink, A. J. and Gifford, D. K. and Jaakkola, T. S. and Young, R. A.},
+  Booktitle                = {Biocomputing 2001},
+  Publisher                = {World Scientific},
+  Year                     = {2000},
+  Pages                    = {422--433}
+}
+
+@Article{Liu2013,
+  Title                    = {Change-Point Detection in Time-Series Data by Relative Density-Ratio Estimation},
+  Author                   = {Liu, S. and Yamada, M. and Collier, N. and Sugiyama, M.},
+  Journal                  = {Neural Networks},
+  Year                     = {2013},
+  Pages                    = {72--83},
+  Volume                   = {43}
+}
+
+@Article{Liu2014,
+  Title                    = {Direct Learning of Sparse Changes in {Markov} Networks by Density Ratio Estimation},
+  Author                   = {Liu, S. and Quinn, J. A. and Gutmann, M. U. and Suzuki, T. and Sugiyama, M.},
+  Journal                  = {Neural Computation},
+  Year                     = {2014},
+  Number                   = {6},
+  Pages                    = {1169--1197},
+  Volume                   = {26}
+}
+
+@Article{Liu2016a,
+  Title                    = {Support Consistency of Direct Sparse-Change Learning in {Markov} Networks},
+  Author                   = {Liu, S. and Suzuki, T. and Relator, R. and Sese, J. and Sugiyama, M. and Fukumizu, K.},
+  Journal                  = {Annals of Statistics},
+  Year                     = {2017},
+
+  Month                    = {06},
+  Number                   = {3},
+  Pages                    = {959--990},
+  Volume                   = {45}
+}
+
+@Book{Mackay2003,
+  Title                    = {Information theory, inference and learning algorithms},
+  Author                   = {MacKay, D.},
+  Publisher                = {Cambridge university press},
+  Year                     = {2003}
+}
+
+@Article{Shimodaira2000,
```

Comment:
@Article{Shimodaira2000Improving,
  author   = {Shimodaira, Hidetoshi},
  title    = {Improving predictive inference under covariate shift by weighting the log-likelihood function},
  journal  = {J. Statist. Plann. Inference},
  year     = {2000},
  volume   = {90},
  number   = {2},
  pages    = {227--244},
  issn     = {0378-3758},
  doi      = {10.1016/S0378-3758(00)00115-4},
  fjournal = {Journal of Statistical Planning and Inference},
  mrclass  = {62B10 (62F15)},
  mrnumber = {1795598},
  url      = {https://doi.org/10.1016/S0378-3758(00)00115-4},
}